### PR TITLE
Support TIFF images.

### PR DESCRIPTION
**Pull request by mikecx**

### CODE REVIEW

**Comment by mikecx** [comment]
I signed it!

File: transform.go [hunk 1]
```diff
@@ -64,7 +65,7 @@ func Transform(img []byte, opt Options) ([]byte, error) {
 		if err != nil {
 			return nil, err
 		}
-	case "jpeg", "webp": // default to encoding webp as jpeg
+	case "jpeg", "webp", "tiff": // default to encoding webp & tiff as jpeg
```
Comment (willnorris) [inline transform.go]: why can't we encode as tiff as well using https://godoc.org/golang.org/x/image/tiff#Encode ?

Comment (mikecx) [reply transform.go]: I suppose we probably can. I was trying to make sure the file would be visible "in-browser" and generally tiff is not supported so it needs to be either jpg or png for that.

Comment (willnorris) [reply transform.go]: Ah, good call.  I didn't realize how poorly supported tiff was in browsers :)

**Comment by willnorris** [comment]
c1a9dab40129cf41be1d7444ab3439deadb76a74 merged.

**Pull request closed without merging**